sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -3395,6 +3398,7 @@ public partial class DiskCreationData
     {
         public DiskCreationData(Azure.ResourceManager.Compute.Models.DiskCreateOption createOption) { }
         public Azure.ResourceManager.Compute.Models.DiskCreateOption CreateOption { get { throw null; } set { } }
+        public string ElasticSanResourceId { get { throw null; } set { } }
```

Comment:
this property should be a resource id (given its name), could you confirm and if so, please assign the `arm-id` format in `rename-mapping` for it

Reply:
addressed